src/fusion_power_plant.h

```diff
@@ -259,6 +259,9 @@ class FusionPowerPlant : public cyclus::Facility  {
   double blanket_turnover;
   double fuel_usage_mass;
   static const double burn_rate; // kg/GW-y
+  double startup_inventory;
+  double tritium_yearly_decay_fraction = 0.055;
```

Comment:
Should this be a `static const` like the burn rate?

Reply:
Probably, yeah.